modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc

```diff
@@ -214,10 +214,9 @@ Configure the execution of the cron job to every 15 min:
 
 [source,bash]
 ----
+
 echo "*/15  *  *  *  * /var/www/owncloud/occ system:cron" \
-  > /var/spool/cron/crontabs/www-data
-chown www-data.crontab /var/spool/cron/crontabs/www-data
-chmod 0600 /var/spool/cron/crontabs/www-data
+  | sudo -u www-data -g crontab tee -a /var/spool/cron/crontabs/www-data
```

Comment:
why not splitting this for better readability into two lines after tee -a ? like `... tee -a \`
long lines are hard to read on the screen